crates/sui-framework/src/natives/crypto.rs

```diff
@@ -40,3 +40,38 @@ pub fn ecrecover(
         Err(_) => Ok(NativeResult::err(cost, INVALID_SIGNATURE)),
     }
 }
+
+pub fn bls12381_verify(
```

Comment:
We should add some docs here mainly explaining that this only works for G1 sigs, G2 keys

Reply:
Yep, for sure - I'll add a doc here mentioning this, but also refer them to crypto.move, which also says this but in greater detail.